src/sql/base/browser/ui/taskbar/overflowActionbar.ts

```diff
@@ -41,8 +41,8 @@ export class OverflowActionBar extends ActionBar {
 		}));
 
 		this._overflow = document.createElement('ul');
-		this._overflow.id = 'overflow';
-		this._overflow.className = 'overflow';
+		this._overflow.id = 'toolbarOverflow';
+		this._overflow.className = 'toolbarOverflow';
```

Comment:
Let's follow standard class naming guidelines and use lowercase, hyphen separated (e.g. `toolbar-overflow`) https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters

Reply:
updated name to toolbar-overflow